nucleus/tile_scheduler/Scheduler.cpp

```diff
@@ -184,6 +184,9 @@ void Scheduler::update_gpu_quads()
                            if (quad.tiles[i].height->size()) {
                                height_data = quad.tiles[i].height.get();
                            }
+                           auto heightimage = nucleus::utils::tile_conversion::toQImage(*height_data);
+                           gpu_quad.tiles[i].height_image = std::make_shared<QImage>(std::move(heightimage));
+                           //TODO: We dont need height in VBO anymore!!! Delete at some point!!
```

Comment:
because you said that it was necessary to have a vbo on some platform ages ago - so i didn't touch it yet. But it's definitely important as I can image it using up some ressources on cpu and gpu alike

Reply:
yes, a vbo is necessary on webgl. but it can be filled with 0. removing height_image would avoid the cpu side storage and code clutter.